### DISCUSSION TITLE
First example sentence for 思う is incorrect

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#18] [Errors and Corrections]
Japanese:

> 明日は天気だと思う。

English:

> I hope it'll be fine tomorrow.

---

The Japanese sentence is pretty much wrong Japanese (I had my doubts but a native speaker confirmed, just in case), the English sentence is fine but is a bit ambiguous without context ("it will be fine" can refer to a lot of things). 

I think it should be changed to:

Japanese:

> 明日はいい天気だといいな

English:

> I hope tomorrow's weather will be good

**Comment by Morgawr** [comment]
Actually I just realized the sentence being changed like that removes the と思う part anyway so... it probably doesn't make sense to use と思う in this kind of example sentence in the first place 🤔

**Comment by stephenmk** [comment]
I'm not sure if the current sentence is technically incorrect. If you search twitter for "明日は天気だと思う" you can find it in a couple of tweets composed by native speakers. I agree that it's definitely not the most natural construction. For an entry as important and common as 思う, I do think it would be better to have a more natural example.

I don't believe we can edit the Japanese sentences in the Tanaka corpus; we're limited to swapping them out for different sentences. Fortunately there's no shortage of sentences for 思う.

I replaced it with this example.
> 晴れだと思うよ。
> I think it will be sunny.

While I'm at it, I added a priority example for sense 03 of 思う ("to judge; to assess; to regard")
> 状況は深刻だと思いますか。
> Do you regard the situation as serious? 

Jim Breen (@JMdictProject) is responsible for the upstream reviewing and approving all example sentence changes each week. He has asked to be kept in the loop particularly for complete sentence changes (as opposed to simple adjustments to the English translations).

**Comment by jimmubreen** [comment]
I try and monitor changes to the J/E sentence pairs but in practice I really only inspect the changes to the Japanese. I have a standing request to be alerted to non-trivial changes. I

My analysis system alerts me to amendments which affect the indices. At present there are about 20-30 of these each week.

**Comment by Morgawr** [comment]
@stephenmk yeah I agree we can just swap out that one sentence anyway for being sketchy even if it might be correct (twitter says there's only 4 posts that are 1:1 verbatim which... usually means it's not really a thing, although 天気 can be used in that sense too). 

@jimmubreen thanks for all the efforts and curating the project upstream. I'm trying to report small fixes here and there, if it's causing problems or there's better ways to do it let me know. I know upstream the tatoeba project seems to focus more on multi-lingual translations (not just J/E) and sometimes it's hard to submit fixes that interfere with other language interpretations.

**Reply by stephenmk** [reply]
No problems that I'm aware of. Tatoeba is organized to keep translations of translations separated from other branches of translations, so I don't think it's causing any interference. If anything I'd assume they're happy to receive more interpretations.

Feel free to continue reporting fixes if you'd like.
